internal/hud/renderer.go

```diff
@@ -74,7 +74,7 @@ func (r *Renderer) layout(v view.View, vs view.ViewState) rty.Component {
 	var ret rty.Component = l
 	if vs.LogModal.TiltLog == view.TiltLogFullScreen {
 		ret = r.renderTiltLog(v, vs, keyLegend(v, vs), ret)
-	} else if vs.LogModal.ResourceLogNumber != 0 {
+	} else if vs.LogModal.ResourceLogNumber != 0 && len(v.Resources) > 0 {
```

Comment:
This doesn't seem like it's correctly solving a problem. What behavior led to this change?

ResourceLogNumber is 1-indexed, so that the default value of 0 means no log.

There's a bug here if you have a log modal open while you delete that log, but that's not a 0/1 issue - if you have resource 4's log open and delete resource 2, you'll now be looking at resource 5 (or get an error if there is no resource 5).

Reply:
Open Tilt with no Tiltfile and press enter; Tilt dies. Would love to prevent this.